redis/src/commands/mod.rs

```diff
@@ -512,6 +512,11 @@ implement_commands! {
         cmd("SISMEMBER").arg(key).arg(member)
     }
 
+    /// Determine if multiple values are members of a set.
+    fn smismember<K: ToRedisArgs, M: ToRedisArgs>(key: K, member: M) {
```

Comment:
minor: please rename member to members, to clarify that the function can take multiple values there.